src/account/account.service.ts

```diff
@@ -120,6 +120,24 @@ export class AccountService {
             .ignore();
     }
 
+    /**
+     * Record an account follow
+     *
+     * @param follower The follower account
+     * @param following This account being followed
+     */
+    async removeAccountFollow(
+        follower: Account,
+        following: Account,
```

Comment:
I think we should switch these around to match `recordAccountFollow` which first takes the account to follow, and then the account that is doing the following.

Also i wonder if `followed` and `follower` could be nicer terms here - i.e "Heres the `followed` account, remove this `follower`" (i suppose `following` also works here)

Reply:
I've switched argument order but kept naming for now for consistency with the db